js/lbank2.js

```diff
@@ -2064,6 +2069,217 @@ module.exports = class lbank2 extends Exchange {
         };
     }
 
+    async fetchDepositWithdrawFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name lbank2#fetchDepositWithdrawFees
+         * @description when using private endpoint, only returns information for currencies with non-zero balance, use public method by specifying this.options['fetchDepositWithdrawFees']['method'] = 'fetchPublicDepositWithdrawFees'
+         * @param {[string]|undefined} codes array of unified currency codes
+         * @param {object} params extra parameters specific to the lbank2 api endpoint
+         * @returns {object} a list of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure}
+         */
+        await this.loadMarkets ();
```

Comment:
Again, shouldn't we fill this up with the "default" network value? 
![image](https://user-images.githubusercontent.com/43336371/204264781-30096bf1-d96f-4eaf-8b40-7f9666aa69dc.png)

Reply:
There's an `isDefault` key, which is checked for, but USDT has no key where `isDefault` is set to true

Reply:
The default is set for withdraw fees, deposit fees aren't provided

Reply:
Yeah, it was at the end of the response, so I didn't notice it; thanks!